### PR TITLE
Create a database based on a diagram

### PR DESCRIPTION
**Pull request by shahira-sadat**
In this PR we did 👇🏻:

- Create a file named `schema_based_on_diagram.sql ` where we implement the database from the diagram
- Join tables from `many-to-many relationships` 
- Add the FK indexes

### CODE REVIEW

File: schema_based_on_diagram.sql [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    total_price DECIMAL(10,2) NOT NULL,
+    invoice_id INT NOT NULL REFERENCES invoices(id),
+    trearment_id INT REFERENCES trearments(id)
+);
+
+
+CREATE INDEX patient_ids ON patient(id);
+CREATE INDEX medical_history_ids ON medical_histories(id);
+CREATE INDEX treatment_ids ON trearments(id);
+CREATE INDEX invoice_ids ON invoices(id);
```
Comment (Whoistolu) [inline schema_based_on_diagram.sql]: - [x] [REQUIRED] Kindly ensure that all FOREIGN KEYS have indexes created properly:
For example: One of the indexes should be `CREATE INDEX ON medical_histories (patient_id)`

**Review by Whoistolu — CHANGES_REQUESTED** [review]
Hi Team
@shahira-sadat and @Fourteen98 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️

- [ ]  _Check the inline comments under the review for sections to improve._

### Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.
### N/A

Cheers and Happy coding!

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @whoistolu in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: schema_based_on_diagram.sql (hunk 1 above)
Comment (shahira-sadat) [reply schema_based_on_diagram.sql]: Done ✅

File: schema_based_on_diagram.sql [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+    invoice_id INT NOT NULL REFERENCES invoices(id),
+    trearment_id INT REFERENCES trearments(id)
+);
+
+CREATE INDEX ON medical_histories (patient_id);
+CREATE INDEX ON invoice_items (invoice_id);
+CREATE INDEX ON invoices (medical_history_id);
+CREATE INDEX ON trearments_medicalhestories (trearment_id);
+CREATE INDEX ON trearments_medicalhestories (medical_history_id);
+CREATE INDEX ON invoice_items (invoice_id);
```
Comment (omar-muhamad) [inline schema_based_on_diagram.sql]: - [x]  [Optional] You have a repeated index in line 56, Please remove it.

**Review by omar-muhamad — APPROVED** [review]
# Approved ✔
### Hi @shahira-sadat and @Fourteen98,

Great job, Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
### Congratulations! 🎉 
### ⚠️Please consider the Optional suggestions below⚠️



## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- Gitflow is used correctly. ✔
- All linters checks are ok. ✔

## (Changes Required) Aspects to improve: ♻️

- N/A

## [Optional] suggestions:

- Please check the inline comments below ⬇.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: schema_based_on_diagram.sql (hunk 2 above)
Comment (shahira-sadat) [reply schema_based_on_diagram.sql]: Thanks, Removed repeated index

**Pull request merged**